gammapy/data/data_store.py

```diff
@@ -711,3 +713,59 @@ def make_psf(self, position, energy=None, theta=None):
         psf_tot = EnergyDependentTablePSF(energy=energy, offset=theta, exposure=exposure,
                                           psf_value=psf_value.T)
         return psf_tot
+
+    def make_mean_edisp(self, position, e_true, e_reco, low_reco_threshold=None, high_reco_threshold=None):
+        r"""Make mean rmf for a given position and a set of observations.
+
+        Compute the mean rmf of a set of observations j at a given position
+
+        The stacking of :math:`j` observations is implemented as follows.  :math:`k`
+        and :math:`l` denote a bin in reconstructed and true energy, respectively.
+
+        .. math::
+
+            \epsilon_{jk} =\left\{\begin{array}{cl} 1, & \mbox{if
+                bin k is inside the energy thresholds}\\ 0, & \mbox{otherwise} \end{array}\right.
+
+            \overline{\mathrm{edisp}}_{kl} = \frac{\sum_{j} \mathrm{edisp}_{jkl}
+                \cdot \mathrm{aeff}_{jl} \cdot t_j \cdot \epsilon_{jk}}{\sum_{j} \mathrm{aeff}_{jl}
+                \cdot t_j}
+
+        Parameters
+        ----------
+        position : `~astropy.coordinates.SkyCoord`
+            Position at which to compute the mean EDISP
+        e_true : `~gammapy.utils.energy.EnergyBounds`
+            True energy axis
+        e_reco : `~gammapy.utils.energy.EnergyBounds`
+            Reconstructed energy axis
+        low_reco_threshold : `~gammapy.utils.energy.Energy`
+            low energy threshold in reco energy, default 0.002 TeV
+        high_reco_threshold : `~gammapy.utils.energy.Energy`
+            high energy threshold in reco energy , default 150 TeV
+        Returns
+        -------
+        stacked_rmf: `~gammapy.irf.EnergyDispersion`
+            Stacked RMF for a set of observation
+        """
+
+        list_arf = list()
+        list_edisp = list()
+        list_livetime = list()
+        if not low_reco_threshold:
+            low_reco_threshold = Energy(0.002, "TeV")
+        if not high_reco_threshold:
+            high_reco_threshold = Energy(150, "TeV")
+        list_low_threshold = [low_reco_threshold] * len(self)
+        list_high_threshold = [high_reco_threshold] * len(self)
+        for obs in self:
```

Comment:
In the following paragraph there are many lines that are longer than the  [recommended length of 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). This makes the code really hard to read. Of course its no problem if a line is sometimes a little bit too long, but if you can avoid it, please do so.  Actually, since you are using pycharm the editor should show you line that are too long and sometimes even give you the option to wrap them automatically (but I don't really know pycharm). In this case you can simply split one long line into several statements, e.g. instead of 
```
list_arf.append(obs.aeff.to_effective_area_table(offset, energy=e_true))
```
```
aeff = obs.aeff.to_effective_area_table(offset=offset,
                                        energy=e_true)
list_arf.append(aeff)
```
For a general style guide for python see https://www.python.org/dev/peps/pep-0008/#introduction, or check out the [pep8 command line tool](https://pypi.python.org/pypi/pep8)